invoke-ai/InvokeAI
# fix(db): retrieve metadata even when no session_id

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

      
## Have you updated all relevant documentation?

## Description

Note: this PR is based on #4105

[fix(db): retrieve metadata even when no session_id](https://github.com/invoke-ai/InvokeAI/commit/73b57a2dce19706650c4b6af778222a29e0b7f07)

this was unnecessarily skipped if there was no `session_id`.

## Related Tickets & Documents

https://discord.com/channels/1020123559063990373/1134237957381947563/1135810973446439033



## QA Instructions, Screenshots, Recordings



Metadata should load for images without a session. This is a fix for @techjedi's import script (on discord).

## Added/updated tests?

- [x] No : nyet

## [optional] Are there any post deployment tasks we need to perform?

nah
